pages/index.tsx

```diff
@@ -8,8 +9,10 @@ import {
   Text,
   Image,
   Input,
+  InputGroup,
+  InputLeftAddon,
+  Center,
 } from "@chakra-ui/react";
-import { motion } from "framer-motion";
 import { useState } from "react";
 
 const theme = {
```

Comment:
Similar to the feedback in yeterday's review, we want to reduce the state usage for these lines
```ts
const [firstColor, setFirstColor] = useState("#000000");
const [secondColor, setSecondColor] = useState("#000000");
```

We can infer the values based on something like a `const [selectedTheme, setSelectedTheme]  = useState()`

and then doing
```ts
const firstColor = theme[selectedTheme][0]
...
```